src/components/sections/courses/PageNavigation.tsx

```diff
@@ -106,16 +106,16 @@ export default function PageNavigation(): ReactElement {
   if (show)
     return (
       <Section>
-        <div className={classNames("text-center justify-center space-x-10 pt-3", { flex: prevUrl, "w-full sm:flex": !prevUrl })}>
+        <div data-testid="pageNavId" className={classNames("text-center justify-center space-x-10 pt-3", { flex: prevUrl, "w-full sm:flex": !prevUrl })}>
           {prevUrl && (
-            <Link href={prevUrl}>
+            <Link href={prevUrl} data-testId="pageNavLinkId">
```

Comment:
same here, access the link using `link` role and `href` content instead of `data-testid`